src/redux/document-content/reducers.ts

```diff
@@ -0,0 +1,15 @@
+import { Reducer } from 'redux'
+import { DocumentContent, DocumentContentAction, DocumentContentActionType, SetDocumentContentAction } from './types'
+
+export const initialState: DocumentContent = {
+  content: ''
```

Comment:
we could move the default content here, if we want.

Reply:
I didn't do that on purpose, because the editor should restore the doc if you close and open the editor.